docs/commands-guide.md

```diff
@@ -1,105 +1,105 @@
 ## Commands Guide
-This is a guide to all of the slash commands for the bot.
+This is a guide to all of the slash commands for the app.
 
 * Action Commands are commands that do not affect a user's `preference file`.
 * Guild Commands can also be considered action commands.
 
 > [!NOTE]
-> Administrator commands are only usable by actually administrators on the Discord server.
+> Administrator commands are only usable by actual administrators on the Discord server.
 
 ### Guild Commands (Administrator)
-1. Disable (or Toggle Chat) Command
-This command will `enable` or `disable` whether or not the bot will respond to users. For example, we set the `enabled` field to `true` to allow the bot to respond to users.
+1. Disable (or Toggle Chat)  
+    This command will `enable` or `disable` whether or not the app will respond to users.  
 
-```
-/toggle-chat enabled true
-```
+    ```
+    /toggle-chat enabled true
+    ```
 
-2. Shutoff Command
-This command will shutoff the bot and no users will be able to respond to the bot anymore. The bot must be manually restarted upon being shutoff.
+2. Shutoff  
+    This command will shutoff the app so no users can converse with it.  
+    The app must be manually restarted upon being shutoff.
 
-Below we shutoff the bot by putting `true` in the `are-your-sure` field.
+    Below shuts off the app by putting `true` in the `are-your-sure` field.
 
-```
-/shutoff are-you-sure true
-```
+    ```
+    /shutoff are-you-sure true
+    ```
 
 ### Action Commands
-1. Clear Channel (Message) History Command
-This command will clear the history of the current channel for the user that calls it. For example, just running the command in any channel will clear the message history.
+1. Clear Channel (Message) History  
+    This command will clear the history of the current channel for the user that calls it.  
+    Running the command in any channel will clear the message history.
 
-```
-/clear-user-channel-history
-```
+    ```
+    /clear-user-channel-history
+    ```
 
-2. Pull Model Command
-This command will pull a model that exists on the [Ollama Model Library](https://ollama.com/library). If it does not exist there, it will throw a hissy fit.
+2. Pull Model  
+    This command will pull a model that exists on the [Ollama Model Library](https://ollama.com/library). If it does not exist there, it will throw a hissy fit.
 
-Below we try to pull the `codellama` model.
+    Below trys to pull the `codellama` model.
 
-```
-/pull-model model-to-pull codellama
-```
+    ```
+    /pull-model model-to-pull codellama
+    ```
 
-3. Thread Create Command
-This command creates a public thread to talk with the bot if you do not want to talk in a `GuildText` channel.
+3. Thread Create  
+    This command creates a public thread to talk with the app instead of using a `GuildText` channel.
 
-You can just run the command below:
+    ```
+    /thread
+    ```
```

Comment:
Is there a reason why these lines (and others) are now indented? Is because of CLRF?

Reply:
Indentation here allows the in-line code blocks to fall under each bullet point instead of at the root of the indentation tree.